internal/removals/remove.go

```diff
@@ -153,6 +153,16 @@ func GetRemainingMap(logger logr.Logger, adminClient fdbadminclient.AdminClient,
 		logger.Info("Exclusions to complete", "remainingServers", remaining)
 	}
 
+	if status.Cluster.RecoveryState.ActiveGenerations > 1 {
+		return nil, fmt.Errorf("too many active generations to run the exclude command to verify a process is fully excluded, current active generations: %d", status.Cluster.RecoveryState.ActiveGenerations)
```

Comment:
Since we are only checking for a non-zero value is that still considered to be too many?

Reply:
We are checking if there is more than one active generation, if there are more than one active generation, the cluster could be in a state where we don't want to run the `exclude` command to verify if processes are excluded. In theory this shouldn't be required, as if the cluster has more than one active generation the previous command will return all processes as not excluded: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/pkg/fdbstatus/status_checks.go#L60-L68

Reply:
But I noticed, it's better to move this check inside the `verifyExcludedProcesses` and only do it if the exclude command would be used.

Reply:
> the cluster could be in a state where we don't want to run the exclude command
Is that because exclude command can be run only if the cluster is fully available (and not going through recovery)? In turn, is that because exclude itself would force a recovery?